docs/PactBackendBenchmarks.md

```diff
@@ -62,6 +62,24 @@ variance introduced by outliers: 98% (severely inflated)
 
 # Phase two: comparing to Pact backends
 
+## Pact Pragmas
+
+```
+fastNoJournalPragmas :: [Pragma]
+fastNoJournalPragmas = [
+  "synchronous = OFF",
```

Comment:
Though I can see why setting `synchronous` is beneficial for speed, are we going to have worry about the safety of this pragma going forward?

Reply:
Until proven otherwise, I want us to be super-optimistic. You can always throw away your pactdb and rebuild.